.github/workflows/windows-build.yml

```diff
@@ -116,8 +134,8 @@ jobs:
         uses: actions/upload-artifact@v3
         with:
           path: |
-            rime-${{ env.git_ref_name }}-${{ runner.os }}-${{ matrix.compiler }}.7z
-            rime-deps-${{ env.git_ref_name }}-${{ runner.os }}-${{ matrix.compiler }}.7z
+            rime-${{ env.git_ref_name }}-${{ runner.os }}-${{ matrix.compiler }}-${{ matrix.arch }}.7z
```

Comment:
The "Create distributable" is missing matrix.arch, and it needs to use the same filename.

Reply:
done.